diff --git a/node_modules/electron-installer-redhat/resources/spec.ejs b/node_modules/electron-installer-redhat/resources/spec.ejs
index e07a8a3..15c722d 100644
--- a/node_modules/electron-installer-redhat/resources/spec.ejs
+++ b/node_modules/electron-installer-redhat/resources/spec.ejs
@@ -24,7 +24,7 @@ print('\n\n\n') }
 
 %>%install
 mkdir -p %{buildroot}/usr/
-cp <%= process.platform === 'darwin' ? '-R' : '-r' %> usr/* %{buildroot}/usr/
+cp -r %{_builddir}/usr/* %{buildroot}/usr/
 
 
 %files
